notebook/agentchat_oai_code_interpreter.ipynb

```diff
@@ -36,7 +36,7 @@
     "    \"OAI_CONFIG_LIST\",\n",
     "    file_location=\".\",\n",
     "    filter_dict={\n",
-    "        \"model\": [\"gpt-3.5-turbo\", \"gpt-35-turbo\", \"gpt-35-turbo-0613\", \"gpt-4\", \"gpt4\", \"gpt-4-32k\"],\n",
+    "        \"model\": [\"gpt-3.5-turbo\", \"gpt-35-turbo\", \"gpt-4\", \"gpt4\", \"gpt-4-32k\"],\n",
```

Comment:
what about using gpt-4-turbo? That is unique in openai endpoints now so it can be understood why there is no azure openai endpoint.
BTW, can openai assistant api be used with all these models ?

Reply:
It works with GPT-4 for me.